sql/core/src/main/scala/org/apache/spark/sql/execution/LocalTableScanExec.scala

```diff
@@ -49,7 +49,10 @@ case class LocalTableScanExec(
     if (rows.isEmpty) {
       sqlContext.sparkContext.emptyRDD
     } else {
-      val numSlices = math.min(unsafeRows.length, sqlContext.sparkContext.defaultParallelism)
+      val numSlices = math.min(
+        unsafeRows.length,
+        conf.filesMinPartitionNum.orElse(sqlContext.conf.defaultParallelism)
```

Comment:
why do we need to look at `conf.filesMinPartitionNum` here? It's not a file scan.